emojiHeatmapper/index.html

```diff
@@ -1,80 +1,66 @@
+        <link rel="stylesheet" href="lib/css/ie10-viewport-bug-workaround.css">
+        <!-- Custom styles for this template -->
+        <link href="lib/css/starter-template.css" rel="stylesheet">
 
-  <!-- Begin emoji-picker Stylesheets -->
-  <link href="./lib/css/emoji.css" rel="stylesheet">
-  <!-- End emoji-picker Stylesheets -->
-
-  <title>Loklak Emoji Map</title>
-</head>
-<body background>
-  <div class="container">
-    <div class="page-header">
-      <h1>Loklak Emoji Map</h1>
-      <h2><small>FOSSASIA</small></h2>
-    </div>
-    <div class="row justify-content-center">
-      <div class="col-md-4">
-        <p class="lead emoji-picker-container">
-          <input type="text" class="form-control textarea-control" placeholder="Search Emoji"
-          data-emojiable="true" id="searchField" data-emoji-input="unicode" maxlength="1">
-        </p>
-      </div>
-      <div class="col-md-4">
-        <div class="btn-group">
-          <button id="searchButton" class='btn btn-success search-button' ng-show="Search" type="button">
-            <span style='padding-left:30px; padding-right:30px;'
-                                  class='glyphicon glyphicon-search'></span>
-          </button>
+    </head>
+    <body>
+        <nav class="navbar navbar-default navbar-fixed-top">
+            <div class="container">
+                <div class="navbar-header">
+                    <button type="button" class="navbar-toggle collapsed"
+                            data-toggle="collapse" data-target="#navbar"
+                            aria-expanded="false" aria-controls="navbar">
+                    <span class="sr-only">Toggle navigation</span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    </button>
+                    <a class="navbar-brand" href="#" class="active">Emoji Heatmapper</a>
+                </div>
+                <!--/.nav-collapse -->
+            </div>
+        </nav>
+        <div class="container">
+            <div class="starter-template">
+                <img src="img/logo.png">
+                <h1>Emoji Heatmapper App</h1>
+                <p class="lead">This is a heatmapper tool which shows the occurences of emojis on
+                the map and also displays the sentiment data.</p>
+            </div>
+            <div class="row">
+                <div class="col-md-3"></div>
+                <div class="col-md-9" style="text-align:center;">
+                    <form class="col-lg-12 hidden-xs ng-pristine ng-valid">
+                        <div class="input-group input-group-lg col-sm-8" style="float:left">
+                            <span class="input-group-btn">
+                            <input type="button" class="btn btn-lg btn-primary" id ="button1"
+                                  value="Emoji Map" onclick="window.location.href='emoji-map.html'">
```

Comment:
this one depends on you :) . anchor tag here would be better instead of input tag